next.config.mjs

```diff
@@ -2,8 +2,20 @@ const basePath = process.env.BASE_PATH || ''
 const infuraApiKey = process.env.INFURA_API_KEY
 const alchemyApiKey = process.env.ALCHEMY_API_KEY
 
+const rpcUrls_1 = (process.env.EL_RPC_URLS_1 &&
+  process.env.EL_RPC_URLS_1.split(',')) || [
+  alchemyApiKey ?? `https://eth-mainnet.alchemyapi.io/v2/${alchemyApiKey}`,
```

Comment:
Error: 
You need && instead of ??, otherwise when key is present url is not made